pkgs/by-name/fr/freerdp/package.nix

```diff
@@ -34,33 +27,88 @@
   libxkbfile,
   wayland,
   wayland-scanner,
-  icu,
   libunwind,
   orc,
-  cairo,
-  cjson,
   libusb1,
+  systemd,
+  buildServer ? true,
+  nocaps ? false,
+  withUnfree ? false,
+
+  withJPEG ? true,
+  libjpeg_turbo,
+
+  withKerberos ? true,
+  libkrb5,
+
+  withURIParser ? stdenv.hostPlatform.isLinux,
+  uriparser,
+
+  withFFmpeg ? true,
+  ffmpeg,
+
+  withOpenH264 ? true,
+  openh264,
+
+  withGSM ? true,
+  gsm,
+
+  withAAC ? true,
+  faad2,
+  faac,
+
+  withOpus ? true,
+  libopus,
+
+  withSoxr ? true,
+  soxr,
+
+  withCairo ? true,
+  cairo,
+
+  withAlsa ? stdenv.hostPlatform.isLinux,
+  alsa-lib,
+
+  withPulseAudio ? stdenv.hostPlatform.isLinux,
   libpulseaudio,
+
+  withPrinting ? true,
   cups,
+
+  withFuse ? stdenv.hostPlatform.isLinux,
+  fuse3,
+
+  withUnicode ? true,
+  icu,
+
+  withSmartCard ? true,
   pcsclite,
+
+  withJSON ? true,
+  cjson,
+
+  withSDL ? true,
   SDL2,
   SDL2_ttf,
   SDL2_image,
-  systemd,
-  libjpeg_turbo,
-  libkrb5,
-  libopus,
-  buildServer ? true,
-  nocaps ? false,
-  withUnfree ? false,
+
+  withWebview ? false,
```

Comment:
All those options are insane. We are never going to validate those on updates and in combination, so something will likely be always broken.